setup.py

```diff
@@ -19,9 +19,20 @@
 tests_require = [
     'pytest',
     'mock',
-    'coverage < 4',
+    'coverage < 4'
 ]
 
+install_requires = [
+    'lxml',
+    'click',
+    'boto3',
+    'requests',
+    'configparser'
+]
+
+if system() == 'Windows':
+    install_requires.append('requests-negotiate-sspi')
```

Comment:
Just out of curiosity, is it possible to use sspi from non-windows machine (let's say linux)?